src/ServicePulse.Host/vue/src/components/monitoring/EndpointDetails.vue

```diff
@@ -314,9 +322,10 @@ function formatGraphDecimal(input, deci) {
 //    }, "MonitoringConnectionStatusChanged");
 //};
 onMounted(() => {
+    getUrlQueryStrings();
   getEndpointDetails();
   getDisconnectedCount(); // for refresh interval
-  console.log(endpoint.value);
+    console.log(endpoint.value);
```

Comment:
do we keep these console.log for now?

Reply:
Yes, these pages are not yet complete, I will remove them later on